spec/channels/application_cable/connection_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
```

Comment:
I'm not sure this line is necessary, I believe this file is included automatically when tests are run